src/packages/@ncigdc/modern_components/environment.js

```diff
@@ -14,12 +19,17 @@ const simpleCache = {};
 const pendingCache = {};
 const handlerProvider = null;
 
-const fetchQuery = addControlledAccessParams => (operation, variables, cacheConfig) => {
+const fetchQuery = (addControlledAccessParams, forceStudyParam) => (operation, variables, cacheConfig) => {
   const body = JSON.stringify({
     query: operation.text, // GraphQL text from input
     variables,
     ...addControlledAccessParams( // This function will also log queries out in dev mode.
-      cacheConfig.requiresStudy ? JSON.stringify(cacheConfig) : '',
+      cacheConfig.requiresStudy
+        ? JSON.stringify(cacheConfig)
+        // Study param from mutations.
```

Comment:
Commented-out bit to enable passing Study params for other queries (e.g. mutations.)
Pending API implementation of that functionality.